tests/data_context/test_data_context_v013.py

```diff
@@ -168,16 +172,17 @@ def test_get_config(empty_data_context):
 
     # We can call get_config in several different modes
     assert type(context.get_config()) == DataContextConfig
-    assert type(context.get_config(mode="typed")) == DataContextConfig
-    assert type(context.get_config(mode="dict")) == dict
+    assert type(context.get_config(mode=ConfigOutputModes.TYPED)) == DataContextConfig
+    assert type(context.get_config(mode=ConfigOutputModes.DICT)) == dict
+    assert type(context.get_config(mode=ConfigOutputModes.YAML)) == str
     assert type(context.get_config(mode="yaml")) == str
```

Comment:
We're adding this enum purely for type safety and better documentation through code (developers using this method know what their options are without having to look at external documentation).

External users should still invoke it with "yaml" - the existing "API" should not change here. My thought process was to test these methods in the way that the user would use them in the notebook. Since they're not using the enum (but rather the string representation), it makes sense to alter our tests to adhere to the user pattern.

As long as we know that "yaml" and "ConfigOutputModes.YAML" are equivalent, I think this is fine.